Yoast/javascript
# Add table css to monorepo.css

## Summary

To avoid duplicate css code in `wordpress-seo`, `table.css` is included in `monorepo.css`.


This PR can be summarized in the following changelog entry:

* Includes table.css in monorepo.css.

## Relevant technical choices:

* 

## Test instructions

This PR can be tested by following these steps:

* See [PR for P3-41](https://github.com/Yoast/wordpress-seo/pull/15673)

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes [P3-41]
